maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,422 @@
+
+            finally:
+                scribe().debug(f"――― End: {base_op_name} ―――")
+
+        return maxim_wrapper
+
+    # Patch AgentWorkflow.run
+    if hasattr(AgentWorkflow, "run"):
+        original_run = AgentWorkflow.run
+        scribe().debug("[MaximSDK] Patching AgentWorkflow.run")
+        wrapper = make_maxim_wrapper(original_run, "llama_index.AgentWorkflow.run")
+        setattr(AgentWorkflow, "run", wrapper)
```

Ruff (0.12.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)